website/static/games/ridgerider_games/minesweeper/minesweeper.css

```diff
@@ -112,10 +177,44 @@ body {
 @keyframes reveal {
     from {
         font-size: 0%;
-        background-color:#707070;
+        background-color:#303030;
+    }
+    20% {
+        background-color: #202020;
+    }
+    50% {
+        font-size: 150%;
     }
     to {
         font-size: 130%;
-        background-color:#909090;
+        background-color:#909090
+    }
+}
+@media only screen and (max-width: 1000px) {
```

Comment:
If you use something like flexbox to position your elements, you won't need media queries except for really unique use-cases because it can automatically resize depending on the screen size.